docs/Demographics.md

```diff
@@ -4,8 +4,8 @@
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
 **age** | **int** | the age of the casualty, omit if unknown | [optional] 
-**sex** | **str** | the sex of the casualty, omit if unknown/indeterminate | [optional] 
-**rank** | **str** | The military status of the casualty, omit if unknown | [optional] 
+**sex** | [**DemographicsSex**](DemographicsSex.md) |  | [optional] 
+**rank** | [**DemographicsRank**](DemographicsRank.md) |  | [optional] 
```

Comment:
We want to be careful as we had a bug in the September Milestone where we conflated "rank" which should be an actual military rank and what we are calling "role" which would be, Marine, Intelligence Officer, etc.  Doesn't necessarily pertain here just noting it.

Reply:
Yep, in the provisional YAML I've written for [ITM-123](https://nextcentury.atlassian.net/browse/ITM-123), that distinction is made in accordance with the [Scenario Design and Variables](https://nextcentury.atlassian.net/wiki/spaces/ITMC/pages/3023372303/Scenario+Design+and+Variables) page.